runtime/moonbase/src/lib.rs

```diff
@@ -204,12 +206,34 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = ();
 }
 
+pub struct DealWithFees<R>(sp_std::marker::PhantomData<R>);
+impl<R> OnUnbalanced<NegativeImbalance<R>> for DealWithFees<R>
+where
+	R: pallet_balances::Config + pallet_treasury::Config,
+	pallet_treasury::Module<R>: OnUnbalanced<NegativeImbalance<R>>,
+{
+	fn on_unbalanceds<B>(mut fees_then_tips: impl Iterator<Item = NegativeImbalance<R>>) {
+		if let Some(fees) = fees_then_tips.next() {
+			// for fees, 80% are burned, 20% to the treasury
+			let (_, to_treasury) = fees.ration(80, 20);
+			// TODO: why support tips? where should tips be sent? substrate impl commented out:
+			// if let Some(tips) = fees_then_tips.next() {
```

Comment:
The TODO comment I am assuming is related to a EIP-1559-alike implementation. Not applicable for us at the moment I guess?

Reply:
The TODO comment is just because I had these questions when I was commenting out the Kusama implementation of `DealWithFees` (handles transaction fees).

On another note, I found it a bit odd that none of the transaction fees nor tips are sent to the block author.

Reply:
me too..